site/src/pages/components/combobox.explainer.mdx

```diff
@@ -0,0 +1,277 @@
+
+This document aims to provide a complete view of combo-box utilisation across the web to inform a potential proposal within Open UI. The goal of this document is to propose a very basic anatomy to simplify the initial shipment of `<combobox>` (or some other name) in the web platform. It will heavily build upon `selectlist` but provide a better accessible and extensible solution than that of an `input` bound with `datalist`.
+
+
+### Out of scope of this document
+
+This document will not be attempting some of the complex scenarios found across the web that leverage various ways in which to represent multi-selected options. Additionally, it does not intend to cover the addition of options that are not available in a `<listbox>`. It will not cover multi-select but will leverage the resolutions defined by `selectlist`.
+
+A good analogy for what this document will result in looking like is the [Tabs research document.](https://open-ui.org/components/tabs.research.parts/)
+
+## Recommended anatomy for version 1
+
+```
+<combobox>
+   <input type=selectlist>
+   <listbox>
+	<option>One</option>
+	<option>Two</option>
+   </listbox>
+</combobox>
+```
+
+
+**Pros:**
+* Clear Semantics: The `<combobox>` element explicitly conveys its purpose.
+
+**Cons:**
+* Introducing new element, `<combobox>`
+
+### Anatomy of `<combobox>`:
+
+<ComboboxAnatomy />
+
+- `<combobox>`: The root container that encapsulates the entire `combobox` structure. It provides context for the interaction between the `input` and the `listbox`.
+  - Slots:
+    - `<input>`
+    - `<listbox>`
+  - Attributes:
+    - **search**: Indicates the combobox will actively fetch results based on user input.
+    - **filter**: Indicates the combobox will narrow down visible options based on user input.
+    - **multiple**: Allows multiple options to be selected.
+- `<input type=selectlist>`: This is where the user can type input or view the selected option. It acts as the trigger to display the listbox.
+  - Slots:
+    - **placeholder** (optional): To display hint text when the input is empty.
+    - **selectedoption** (optional): To display the currently selected option from the `listbox`. 
+      - Note: Currently input doesn't support slots
+- `<listbox>`: This container holds the selectable options. It's hidden by default and is displayed when the `<input type=selectlist>` is activated.
+  - Slots:
+    - `<option>`
+    - `<optgroup>` (optional)
+- `<option>`: Represents an individual selectable item within the `listbox`.
+- `<optgroup>` (Optional): A container to group related options together.
+  - Slots:
+    - `<legend>`: To label the group.
+    - `<option>`: The options within this group.
+- `<legend>` (Optional, within `<optgroup>`): Provides a label or title for a group of options within the `listbox`.
+
+### Default Behavior
+
+Just like a regular dropdown, the `<combobox>` doesn't need much to get started. Here's how you can set it up with the basics:
```

Comment:
Do not use the term `dropdown` anywhere in this explainer. If you want to use this introduction text in this sentence then reference `selectlist` and link to its anatomy.